stdlib/_operator.pyi

```diff
@@ -105,22 +108,10 @@ class attrgetter(Generic[_T_co]):
 
 @final
 class itemgetter(Generic[_T_co]):
-    # mypy lacks support for PEP 646 https://github.com/python/mypy/issues/12280
-    # So we have to define all of these overloads to simulate unpacking the arguments
     @overload
-    def __new__(cls, item: _T_co) -> itemgetter[_T_co]: ...
+    def __new__(cls, __item: _T_co) -> itemgetter[_T_co]: ...
```

Comment:
Using `_T_co` in this overload has the same issues when it comes to ambiguous scoping -- though it's less clear how it could cause actual _problems_ when it comes to this overload, it would be nice to use a different TypeVar here as well, just to avoid any ambiguity: 

```suggestion
    def __new__(cls, __item: _T) -> itemgetter[_T]: ...
```